models/classes/taskQueue/TaskLog/Broker/RdsTaskLogBroker.php

```diff
@@ -15,102 +15,68 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2017 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
  *
  */
 
 namespace oat\tao\model\taskQueue\TaskLog\Broker;
 
+use common_persistence_sql_SchemaManager;
+use common_persistence_SqlPersistence as SqlPersistence;
+use common_persistence_Persistence as Persistence;
 use Doctrine\DBAL\Connection;
-use oat\oatbox\PhpSerializable;
-use common_report_Report as Report;
+use Doctrine\DBAL\ParameterType;
 use Doctrine\DBAL\Query\QueryBuilder;
+use common_report_Report as Report;
+use oat\generis\persistence\PersistenceManager;
 use oat\tao\model\taskQueue\QueueDispatcherInterface;
 use oat\tao\model\taskQueue\Task\CallbackTaskInterface;
 use oat\tao\model\taskQueue\Task\TaskInterface;
 use oat\tao\model\taskQueue\TaskLog\CategorizedStatus;
 use oat\tao\model\taskQueue\TaskLog\CollectionInterface;
 use oat\tao\model\taskQueue\TaskLog\Entity\EntityInterface;
-use oat\tao\model\taskQueue\TaskLog\TaskLogCollection;
 use oat\tao\model\taskQueue\TaskLog\TaskLogFilter;
 use oat\tao\model\taskQueue\TaskLog\TasksLogsStats;
 use oat\tao\model\taskQueue\TaskLogInterface;
-use Psr\Log\LoggerAwareInterface;
-use Zend\ServiceManager\ServiceLocatorAwareTrait;
 use oat\oatbox\log\LoggerAwareTrait;
 
 /**
  * Storing message logs in RDS.
  *
  * @author Gyula Szucs <gyula@taotesting.com>
  */
-class RdsTaskLogBroker implements TaskLogBrokerInterface, PhpSerializable, LoggerAwareInterface, RdsTaskLogBrokerInterface
+class RdsTaskLogBroker extends AbstractTaskLogBroker
 {
-    use ServiceLocatorAwareTrait;
     use LoggerAwareTrait;
 
+    /** @var string */
     private $persistenceId;
 
-    /**
-     * @var \common_persistence_SqlPersistence
-     */
+    /** @var SqlPersistence */
     protected $persistence;
 
-    private $containerName;
-
     /**
      * RdsTaskLogBroker constructor.
      *
      * @param string $persistenceId
      * @param null $containerName
      */
-    public function __construct($persistenceId, $containerName = null)
+    public function __construct(string $persistenceId, ?string $containerName = null)
```

Comment:
```suggestion
    public function __construct(string $persistenceId, string $containerName = null)
```
A default value is already being provided